contracts/standard/NonFungibleToken.cdc

```diff
@@ -4,8 +4,8 @@
 
 ## `NonFungibleToken` contract interface
 
-The interface that all non-fungible token contracts could conform to.
-If a user wants to deploy a new nft contract, their contract would need
+The interface that all Non-Fungible Token contracts could conform to.
```

Comment:
Is this an update for this file from mainnet/testnet? It should be in sync with those

Reply:
yes, this is from mainnet